src/index.js

```diff
@@ -1,122 +1,157 @@
+  const $ = await request({
+    uri: `${baseURL.origin}/fr/orders`,
+    transform: body => cheerio.load(body),
+    jar: cookiejar
+  })
   log('info', 'Parsing list of documents')
   const documents = await parseDocuments($)
 
-  // here we use the saveBills function even if what we fetch are not bills, but this is the most
-  // common case in connectors
   log('info', 'Saving data to Cozy')
-  await saveBills(documents, fields.folderPath, {
-    // this is a bank identifier which will be used to link bills to bank operations. These
-    // identifiers should be at least a word found in the title of a bank operation related to this
-    // bill. It is not case sensitive.
-    identifiers: ['books']
-  })
-}
-
-// this shows authentication using the [signin function](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_signin)
-// even if this in another domain here, but it works as an example
-function authenticate(username, password) {
-  return signin({
-    url: `http://quotes.toscrape.com/login`,
-    formSelector: 'form',
-    formData: { username, password },
-    // the validate function will check if
-    validate: (statusCode, $) => {
-      // The login in toscrape.com always works excepted when no password is set
-      if ($(`a[href='/logout']`).length === 1) {
-        return true
-      } else {
-        // cozy-konnector-libs has its own logging function which format these logs with colors in
-        // standalone and dev mode and as JSON in production mode
-        log('error', $('.error').text())
-        return false
+  await saveBills(
+    documents.map(entry => ({
+      ...entry,
+      requestOptions: {
+        jar: cookiejar
       }
+    })),
+    fields.folderPath,
+    {
+      identifiers: ['deliveroo']
     }
+  )
+}
+
+async function authenticate(username, password) {
+  const options = {
+    formselector: 'form#login',
+    input: {
+      login: 'login_email',
+      password: 'login_password'
+    },
+    username,
+    password,
+    generateHeader: $ => ({
+      'x-csrf-token': $('meta[name="csrf-token"]').attr('content'),
+      'content-type': 'application/json;charset=UTF-8'
+    }),
+    validate: response => response && JSON.parse(response.body).user.id,
+    debug: false
+  }
+
+  return await signin(options)
+}
+
+async function signin({
+  formselector = 'form',
+  input = { login: 'email', password: 'password' },
+  username,
+  password,
+  generateHeader = () => ({}),
+  validate = () => false
+}) {
+  const loginurl = baseURL.href
+  const loginpage = await request({
+    uri: loginurl,
+    resolveWithFullResponse: true,
+    transform: (body, response, resolveWithFullResponse) =>
+      resolveWithFullResponse
+        ? { ...response, parsedBody: cheerio.load(body) }
+        : cheerio.load(body),
+    jar: cookiejar
+  }).catch(err => {
+    console.error(err)
```

Comment:
Be carefull, these errors are not shown in the logs in production. This should be a : 

```javascript
log('error', err)
```